js/zb.js

```diff
@@ -1753,13 +1755,23 @@ module.exports = class zb extends Exchange {
         });
         const request = {
             'amount': this.amountToPrecision (symbol, amount),
+            // 'acctType': 0, // Spot, Margin 0/1/2 [Spot/Isolated/Cross] Optional, Default to: 0 Spot
+            // 'customerOrderId': '1f2g', // Spot, Margin
+            // 'orderType': 1, // Spot, Margin order type 1/2 [PostOnly/IOC] Optional
         };
         if (price) {
             request['price'] = this.priceToPrecision (symbol, price);
         }
         if (spot) {
             request['tradeType'] = (side === 'buy') ? '1' : '0';
             request['currency'] = market['id'];
+            if (timeInForce === 'PO') {
+                request['orderType'] = 1;
+            } else if (timeInForce === 'IOC') {
+                request['orderType'] = 2;
+            } else {
+                request['orderType'] = undefined;
```

Comment:
Let's not send the orderType param if it is optional and if timeInForce is undefined.

Reply:
The rule of thumb is: if a parameter is optional and not specified by the user – we don't send it.